app/services/recaptcha_validator.rb

```diff
@@ -75,17 +92,18 @@ def recaptcha_score_meets_threshold?(score)
     end
   end
 
-  def recaptcha_errors_exempt?(error_codes)
-    (error_codes - EXEMPT_ERROR_CODES).blank?
+  def is_result_error?(error_code)
+    RESULT_ERRORS.include?(error_code)
   end
 
-  def log_analytics(recaptcha_result: nil, error: nil)
+  def log_analytics(result: nil, error: nil)
     analytics&.recaptcha_verify_result_received(
-      recaptcha_result:,
+      recaptcha_result: result.presence && result.to_h,
```

Comment:
`nil.to_h == {}` so we could probably do:
```suggestion
      recaptcha_result: result.to_h.presence,
```

Reply:
> `nil.to_h == {}` so we could probably do:

That's pretty clever!